pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/DocumentService.java

```diff
@@ -567,7 +569,7 @@ public DocumentsMetaData createDocumentResponse(List<DocumentEntity> entityList)
 		List<DocumentMultipartResponseDTO> allDocRes = new ArrayList<>();
 		DocumentsMetaData documentsMetaData = new DocumentsMetaData();
 		for (DocumentEntity doc : entityList) {
-			log.info("","","","Demographic preid:" + doc.getDemographicEntity().getStatusCode());
+			log.info("", "", "", "Demographic preid:" + doc.getDemographicEntity().getStatusCode());
```

Comment:
use , instead of + and also add pre reg id.

Reply:
print prid instead of statuscode